### PR TITLE
product controller updated

### PR DESCRIPTION
**Pull request by Joelle5**
```
on pages#index

.container
  .row
   -@products.each do |product|
    .card
        li = product.name
```

### CODE REVIEW

File: spec/controllers/products_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe ProductsController, type: :controller do
+
+  describe 'GET #edit'do
+
+    let(:product) { create(:product) }
+      before { get :edit, params: { id: product } }
+      xit { expect (assigns(:product)).to eq(product) }
```
Comment (shirlaine) [inline spec/controllers/products_controller_spec.rb]: oops, small typo..

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  def index
+    @products = Product.all
+  end
+
+  def new
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.create(product_params)
```
Comment (shirlaine) [inline app/controllers/products_controller.rb:12]: will change this later to: 
if save, redirect else render new

File: app/controllers/products_controller.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  def edit
+    @product = Product.find(params[:id])
+  end
+
+  def update
+    @products = Product.all
+    @product = Product.find(params[:id])
+    if
+      @product.update_attributes(post_params)
```
Comment (shirlaine) [inline app/controllers/products_controller.rb:24]: @product.update(product_params) 😅

File: spec/controllers/products_controller_spec.rb (hunk 1 above)
Comment (shirlaine) [reply spec/controllers/products_controller_spec.rb]: nice specs 👍

**Review by shirlaine — APPROVED** [review]
can change small typos on product#update and rspec
good to merge after! 🚢

File: spec/controllers/products_controller_spec.rb (hunk 1 above)
Comment (Joelle5) [reply spec/controllers/products_controller_spec.rb]: xit just pend the code, I just got it working already

**Pull request merged**